mkdocs.yml

```
@@ -182,6 +182,7 @@ nav:
                 - App.py: reference-docs/app/app.md
                 - Python Version: reference-docs/python.md
                 - Scaling: reference-docs/app/scaling.md
+                - Settings: reference-docs/app/settings.md
```

The referred markdown file name is not correct I think? 

```suggestion
                - Settings Overview: reference-docs/app/settings_overview.md
```